h/links.py

```diff
@@ -5,7 +5,23 @@
 """
 
 
-from h._compat import urlparse
+from h._compat import urlparse, url_unquote
+
+
+def pretty_link(url):
+    """
+    Return a nicely formatted version of a URL.
+
+    This strips off 'visual noise' from the URL including common schemes
+    (HTTP, HTTPS), domain prefixes ('www.') and query strings.
+    """
+    parsed = urlparse.urlparse(url)
+    if parsed.scheme not in ['http', 'https']:
+        return url
+    netloc = parsed.netloc
+    if netloc.startswith('www.'):
+        netloc = netloc[4:]
+    return url_unquote(netloc + parsed.path)
```

Comment:
This code is a low-level utility that formats URLs for display. The presenters might use this code if they have a need to format links in the same way, though I don't think they do currently.